tests/Baseline/spicy.types.struct.init-fail/output

```diff
@@ -0,0 +1,4 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+[error] <...>/init-fail.spicy:12:25-13:22: declaration needs a concrete struct type
```

Comment:
Unrelated to this change, but that error location is a bit off.

Reply:
Not sure it is exactly the same issue, but I linked this in #1050.